checkstyle/suppressions.xml

```diff
@@ -98,6 +98,7 @@
     <suppress checks="CyclomaticComplexity" files="com/hazelcast/config/AbstractXmlConfigHelper"/>
     <suppress checks="ClassDataAbstractionCoupling" files="com/hazelcast/config/AbstractXmlConfigHelper"/>
     <suppress checks="FanOutComplexity" files="com/hazelcast/config/ConfigXmlGenerator"/>
+    <suppress checks="ClassFanOutComplexityCheck" files="com/hazelcast/config/Config"/>
 
```

Comment:
Hmm, than the other suppression might not work. The shorter name should be correct:
http://checkstyle.sourceforge.net/config_metrics.html#ClassFanOutComplexity

The longer name is the name of the class which defines the check.